### DISCUSSION TITLE
可以把得意黑添加到 iconfont.cn 吗？

### DISCUSSION THREAD

**Discussion by yisibl** [atelier-anchor/smiley-sans#109] [Ideas]
你好，我是 iconfont.cn 的官方开发者。得意黑是不可多得的高质量的中文斜体，受到了社区广泛的好评。在 iconfont 中也收到了希望可以添加得意黑的 issue：https://github.com/thx/iconfont-plus/issues/2414

iconfont 不仅可以提供字体的预览和展示，还可以提供高质量的子集化服务，可以为得意黑在网页展示提供良好的服务。

特此，希望可以得到得意黑官方许可，非常感谢。

**Comment by yisibl** [comment]
@stone-zeng 打扰了，可以帮忙看看吗？

**Reply by stone-zeng** [reply]
可以，只要遵守 license 即可。
